build/.vsts-ci.yml

```diff
@@ -269,7 +269,7 @@ phases:
     condition: always()
 - phase: wolfssl
   variables:
-    _PREVIEW_VSTS_DOCKER_IMAGE: "aziotbld/vsts-c-wolfssl"
+    _PREVIEW_VSTS_DOCKER_IMAGE: "aziotbld/vsts-c-wolfssl:18.04"
```

Comment:
Hey @jebrando ,
should we add this? Wasn't the code getting the latest version anyway before?
Could you share details about why we need to tie it to 18.04 specifically?